src/builderUtils/converters.ts

```diff
@@ -116,15 +117,15 @@ export const schema2value = (
 
     if (schema.oneOf || schema.allOf || schema.anyOf) {
       hasOf = schema.oneOf ? 'oneOf' : schema.allOf ? 'allOf' : 'anyOf'
-      value = of2Values(schema)
+      value = of2Values(schema, required)
     } else if (schema.enum) {
       isEnum = true
       value = schema.type === 'string' ? schema.enum.map(e => `'${e}'`) : schema.enum
     } else if (isArraySchema(schema)) {
       isArray = true
-      value = schema2value(schema.items)
+      value = schema2value(schema.items, required)
     } else if (schema.properties || schema.additionalProperties) {
-      value = object2value(schema)
+      value = object2value(schema, required)
```

Comment:
Fortunately, we can solve the problem just by replacing these `required` to `false`.